project/MiMa.scala

```diff
@@ -1024,7 +1024,11 @@ object MiMa extends AutoPlugin {
 
         // #20553 Tree flattening should be separate from Fusing
         ProblemFilters.exclude[MissingClassProblem]("akka.stream.Fusing$StructuralInfo"),
-        ProblemFilters.exclude[MissingClassProblem]("akka.stream.Fusing$StructuralInfo$")
+        ProblemFilters.exclude[MissingClassProblem]("akka.stream.Fusing$StructuralInfo$"),
+
+        // #21894 Programmatic configuration of the ActorSystem
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this"),
```

Comment:
shouldn't we start a new "2.4.15" section? or is it "2.4.14" - I never remember, and we always forget that after releasing